phpBB/styles/prosilver/template/posting_body.html

```diff
@@ -1,6 +1,11 @@
-
 <!-- IF S_PRIVMSGS -->
 	<!-- INCLUDE posting_pm_layout.html -->
 <!-- ELSE -->
+	<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/jquery.caret.min.js -->
+	<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/jquery.atwho.min.js -->
+	<script type="text/javascript">var U_AJAX_MENTION_URL = '{UA_AJAX_MENTION_URL}'</script>
```

Comment:
Move all of this code to the posting_layout.html, the INCLUDE_CSS can be above the INCLUDE of overall_header.html, the rest after the INCLUDE. this should solve the functional test errors.